src/view/channel-create.js

```diff
@@ -33,7 +33,7 @@ const ChannelCreateView = ({ store, nav, channel }) => (
     <Header separator>
       <Button disabled onPress={() => {}} />
       <Title title="Create Channel" />
-      <CancelButton onPress={() => nav.goChannels()} />
+      <CancelButton onPress={() => nav.goBack()} />
```

Comment:
that's a nice feature, feel like it eliminates a class of potential mistakes :+1: